src/Core/Appplication/Service/ConverterResolver.php

```diff
@@ -29,23 +29,24 @@ public function __construct(ConverterRegistryInterface $converterRegistry)
     /**
      * @param \SprykerSdk\SdkContracts\Entity\CommandInterface $command
      *
-     * @return \SprykerSdk\SdkContracts\Violation\ViolationConverterInterface|null
+     * @return \SprykerSdk\SdkContracts\Report\ReportConverterInterface|null
      */
-    public function resolve(CommandInterface $command): ?ViolationConverterInterface
+    public function resolve(CommandInterface $command): ?ReportConverterInterface
     {
-        $converter = $command->getViolationConverter();
+        $converter = $command->getConverter();
```

Comment:
I think the name $converter in this case is not defining the actual meaning of the object. It consist of name and config only, so its more like converter settings maybe?